### DISCUSSION TITLE
How to use consola in nitro?

### DISCUSSION THREAD

**Discussion by XStarlink** [nitrojs/nitro#1758] [Q&A]
Hi, I'm trying to use consola in nuxt 3 server routes, but the log styles don't display, I just have white text, like with a regular console.log. How can I access consola?

My current code:
```js
import { consola } from 'consola'

export default defineEventHandler(async (event : H3Event) => {
    consola.info('Using consola 3.0.0')
    consola.start('Building the project...')
    consola.warn('A new version of consola is available: 3.0.1')
    consola.success('Project built!')
    consola.error(new Error('This is an example error. Everything is fine!'))
    consola.box('I am a simple box')
})
```

It would be awesome if we could have a utility like this:
```js
const consola = useConsola()
```

**Comment by Davounet** [comment]
I have the same problem and cant get around it !

**Comment by kiki-kanri** [comment]
In nuxt, building a consola logger with useLogger.

```typescript
import { useLogger } from '@nuxt/kit';

const logger = useLogger();
logger.info('Info log.');
```

**Reply by pi0** [reply]
(off topic) please consider to never import `@nuxt/kit` in runtime code. It is purely for build namespace (modules and config)

**Reply by kiki-kanri** [reply]
Sorry for suggesting the wrong solution, thanks for pointing out my mistake!

**Comment by DaDlugosch** [comment]
The same problem.

**Comment by gkTim** [comment]
Also having this problem. Tried the default and createConsola method with fancy flag set. Anyone found a solution?

**Comment by jsonleex** [comment]
In my project, I use consola like this 👇

```ts
// utils/logger.ts
import { consola } from 'consola'
import { env } from 'node:process'

if (env.NODE_ENV === 'development') {
  env.CONSOLA_LEVEL = 'info'
}

export const logger = consola.withTag('YourAppName')
```
